lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -43,6 +43,8 @@ namespace mlir::tt::ttir {
 #define GEN_PASS_DEF_TTIROPTIMIZER
 #define GEN_PASS_DEF_TTIRIMPLICITDEVICE
 #define GEN_PASS_DEF_TTIRLOADSYSTEMDESC
+#define GEN_PASS_DEF_TTIRGATHERPATTERNMATCH
+#include "ttmlir/Conversion/Passes.h.inc"
```

Comment:
Once you move `TTIRGatherPatternMatch` class to `include/ttmlir/Dialect/TTIR/Transforms/Passes.td`; you don't need to add `#include "ttmlir/Conversion/Passes.h.inc"`.